src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -72,6 +74,17 @@ const defaultProps = {
 };
 
 class ReimbursementAccountPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.continue = this.continue.bind(this);
+
+        const achData = lodashGet(this.props, 'reimbursementAccount.achData', {});
+        const hasInProgressVBBA = achData.bankAccountID && achData.state !== BankAccount.STATE.OPEN;
+        this.state = {
+            isReadyToContinue: !hasInProgressVBBA,
```

Comment:
I find this variable name (isReadyToContinue) confusing. Maybe rename to `shouldShowContinueSetupButton` or `shouldShowStartBackView` (and therefore inverse the logic)